From a4fea77b144995d5ae533c8d3b0efdb959dc63cb Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 10 Jan 2014 09:36:24 -0500
Subject: [PATCH 34/34] Fix grub_script_execute_sourcecode() usage on ppc.

593e430c made it not take the extra argc/argv that this code still
passes it.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 grub-core/normal/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grub-core/normal/main.c b/grub-core/normal/main.c
index 6c24a07..6eae828 100644
--- a/grub-core/normal/main.c
+++ b/grub-core/normal/main.c
@@ -281,7 +281,7 @@ grub_normal_execute (const char *config, int nested, int batch)
       if (! grub_ieee1275_cas_reboot (script))
         {
           char *dummy[1] = { NULL };
-          if (! grub_script_execute_sourcecode (script, 0, dummy))
+          if (! grub_script_execute_sourcecode (script))
             boot = 1;
         }
       grub_free (script);
-- 
1.8.4.2

